recipes/pylance/meta.yaml

```diff
@@ -28,6 +28,7 @@ requirements:
     - numpy >=1.22
     - pyarrow >=12
     - pyarrow-hotfix
```

Comment:
We probably won't support installing `pyarrow-hotfix` with newer arrow versions. You should omit it here.
```suggestion
```